playbooks/roles/unbound/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Install unbound and resolvconf
+  apt:
+    name={{ item }}
+    state=installed
+  with_items:
+    - unbound
+    - resolvconf
+
+- name: Copy the configuration for caching and forwarding
+  template:
+    src: cache-and-forward.conf.j2
+    dest: "{{ unbound_config_dir }}/cache-and-forward.conf"
```

Comment:
Is `unbound_config_dir` already created upon installation above?

Reply:
@UmanShahzad, yes it is.

```
# dpkg -L unbound
/.
/etc
/etc/apparmor.d
/etc/apparmor.d/usr.sbin.unbound
/etc/default
/etc/init.d
/etc/init.d/unbound
/etc/insserv.conf.d
/etc/insserv.conf.d/unbound
/etc/resolvconf
/etc/resolvconf/update.d
/etc/resolvconf/update.d/unbound
/etc/unbound
/etc/unbound/unbound.conf
/etc/unbound/unbound.conf.d
/etc/unbound/unbound.conf.d/qname-minimisation.conf
/etc/unbound/unbound.conf.d/root-auto-trust-anchor-file.conf
/lib
/lib/systemd
/lib/systemd/system
/lib/systemd/system/unbound-resolvconf.service
/lib/systemd/system/unbound.service
/usr
/usr/lib
/usr/lib/resolvconf
/usr/lib/resolvconf/dpkg-event.d
/usr/lib/resolvconf/dpkg-event.d/unbound
/usr/lib/unbound
/usr/lib/unbound/package-helper
/usr/sbin
/usr/sbin/unbound
/usr/sbin/unbound-checkconf
/usr/sbin/unbound-control
/usr/sbin/unbound-control-setup
/usr/share
/usr/share/doc
/usr/share/doc/unbound
/usr/share/doc/unbound/CREDITS
/usr/share/doc/unbound/FEATURES
/usr/share/doc/unbound/NEWS.Debian.gz
/usr/share/doc/unbound/README.DNS64
/usr/share/doc/unbound/README.gz
/usr/share/doc/unbound/TODO.gz
/usr/share/doc/unbound/changelog.Debian.gz
/usr/share/doc/unbound/changelog.gz
/usr/share/doc/unbound/contrib
/usr/share/doc/unbound/contrib/update-anchor.sh.gz
/usr/share/doc/unbound/copyright
/usr/share/doc/unbound/examples
/usr/share/doc/unbound/examples/unbound.conf
/usr/share/man
/usr/share/man/man5
/usr/share/man/man5/unbound.conf.5.gz
/usr/share/man/man8
/usr/share/man/man8/unbound-checkconf.8.gz
/usr/share/man/man8/unbound-control.8.gz
/usr/share/man/man8/unbound.8.gz
/usr/share/munin
/usr/share/munin/plugins
/usr/share/munin/plugins/unbound_munin_
/var
/var/lib
/var/lib/unbound
```